src/components/HypothesisBuilder/components/HypothesisNodeNew/index.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Disclosure } from "@headlessui/react";
+import { ChevronDownIcon } from "@heroicons/react/solid";
+import cx from "clsx";
+import { Icon } from "../../../Icon";
+import { HypothesisNodeItem } from "./HypothesisNodeItem";
+
+export const HypothesisAccordion = ({
+  node,
+  parentArray,
+  setModalIsOpen,
+  setSelectedNodePath
+}) => {
+  const handleOpenModal = () => {
+    setSelectedNodePath([...parentArray, node.id]);
+    setModalIsOpen(true);
+  };
+  const getStatusIcon = (status) => {
+    switch (status) {
+      case "proven":
+        return "like";
+      case "possible":
+        return "search";
+      case "disproven":
+        return "dislike";
+      default:
+        return "like";
+    }
+  };
+
+  return (
+    <div>
+      <div className="flex align-baseline text-base font-semibold mb-6">
+        <h2 className="text-dark-gray mr-3 font-inter text-2xl">Action plan</h2>
+        <div className="bg-dark-blue rounded-full p-2">
+          <Icon name="addButton" className="w-3 h-3" />
+        </div>
+      </div>
+      <div className="w-full mt-5">
+        <div className="w-full bg-white rounded-8px font-inter">
+          <Disclosure key={node.id}>
+            {({ open }) => (
+              <>
+                <Disclosure.Button className="w-full flex justify-between">
+                  <HypothesisNodeItem
+                    renderChevronIcon={() => (
+                      <ChevronDownIcon
+                        className={cx(
+                          "w-5 h-5 ml-4 mr-2",
+                          open ? "transform rotate-180" : ""
+                        )}
+                      />
+                    )}
+                    iconLeft={getStatusIcon(node.status)}
+                    text={node.title}
+                    iconRightFirst="message"
+                    iconRightSecond="diagram"
+                    image="https://images.unsplash.com/photo-1531427186611-ecfd6d936c79?ixlib=rb-=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=8&w=256&h=256&q=80"
```

Comment:
This component is already wired to the backend, please don't introduce dummy data.

Reply:
where I can get data for avatars and this icons? from backend i get this data
<img width="477" alt="Screenshot 2022-02-02 at 3 32 15 PM" src="https://user-images.githubusercontent.com/32077311/152163607-5a9d220f-b320-441a-9aa7-6d12b1f534a4.png">

Reply:
Updating the getHypothesis query to `/db/incident?id=$incidentId&select=*,hypothesis!hypothesis_incident_id_fkey(*,created_by(name,avatar)),commander_id(name,avatar),communicator_id(name,avatar)` should work

Reply:
Hey @moshloop . First of all, i dont think it's correct to fetch incident entity in the function which is named "getHypothesis". And in any case your suggestion did not work very well. I decided that we can alter getIncident so it returns name and avatars of creators also. Is it ok ?

```
export const getIncident = async (id) =>
  resolve(
    IncidentCommander.get(
      `/incident?id=eq.${id}&select=*,hypothesis!hypothesis_incident_id_fkey(*,created_by(name,avatar))`
    )
  );
```

I looked at the response - avatar values contain "string" and null values. Is it expected?

Also can we get some clarifications about these "message" and "signal" icons?
![image](https://user-images.githubusercontent.com/20907048/152344217-1035dacc-c65b-427a-9339-58e165f60d80.png)

Where can we get data to know if they should be visible, etc...

Reply:
You can rename getHypothesis to getIncident - however there is a created_by, commander, communicator that needs to be shown in the details section (not under the action plan)

For each hypothesis there can be mulltiple icons:

- creator of the hypothesis
- all responders/commentators in the hypothesis (not currentle returned by api)
- a message/comment icon to indicate there are comments
- a log icon to indicate there are log evidence types associated (and the same for metrics and traces)


The avatar can be null in which case use a Circle with initials

Reply:
@moshloop Can we clarify what data will be available in the future and what can be obtained now, where and how?

> creator of the hypothesis

it's avator of `"created_by":{"name":"System","avatar":null}` piece of json, as i understand
> all responders/commentators in the hypothesis (not currentle returned by api)

will "incident" route return this data, or maybe we will have to make "comments" request? anyway, any examples or description of what it would look like if the hypothesis had 10 unique commentators?
> a message/comment icon to indicate there are comments

does api return this data at the moment? please describe it in more details because it's hard to work without api documentation and a complete understanding of the subject area.
> a log icon to indicate there are log evidence types associated (and the same for metrics and traces)

![image](https://user-images.githubusercontent.com/20907048/153215679-39666e6c-04b7-425f-8bd7-7ae9a0d5451e.png)
is this log icon? does api return this data at the moment?

Reply:
@moshloop 

> that needs to be shown in the details section

1. what is details section at all? it is somewhere in the modal?
2. do i correctly understand that this task/PR should introduce new "Add hypothesis" modal and upgraded "Edit hypothesis" modal ?